sched/scheduler/stateful_scheduler.go

```diff
@@ -321,6 +316,20 @@ func (s *statefulScheduler) step() {
 	s.checkForCompletedJobs()
 	s.killJobs()
 	s.scheduleTasks()
+
+	remaining := 0
+	waitingToStart := 0
+	for _, job := range s.inProgressJobs {
+		remaining += (len(job.Tasks) - job.TasksCompleted)
+		log.Infof("***jobid:%s, completed:%d, running:%d", job.Job.Id, job.TasksCompleted, job.TasksRunning)
```

Comment:
I suspect this is too verbose and I believe this information is already printed after doing scheduleTasks()